docs/reference/tab-widgets/troubleshooting/data/total-shards-per-node.asciidoc

```diff
@@ -0,0 +1,191 @@
+//////////////////////////
+
+[source,console]
+--------------------------------------------------
+PUT my-index-000001
+{
+  "settings": {
+    "index.routing.allocation.total_shards_per_node": "1"
+  }
+}
+
+--------------------------------------------------
+// TESTSETUP
+
+[source,console]
+--------------------------------------------------
+DELETE my-index-000001
+--------------------------------------------------
+// TEARDOWN
+
+//////////////////////////
+
+// tag::cloud[]
+In order to get the shards assigned we'll need to increase the number of shards 
+that can be collocated on a node. 
+We'll achieve this by inspecting the configuration for the `index.routing.allocation.total_shards_per_node` 
+<<indices-get-settings, index setting>> and increasing the configured value for the
+indices that have shards unassigned.
+
+Note that this requires the current data nodes to have enough disk space to host 
+all the data.
+If the current deployment configuration doesn't have enough capacity to store
+all the data, including the unassigned shards, please contact 
+https://support.elastic.co[Elastic Support] to discuss available options.
```

Comment:
Similar to Andrei's [comment above](https://github.com/elastic/elasticsearch/pull/86214/files#r867867753), I'd remove these lines in the Note as this is a setting to control cluster balance.  Disk usage is a different matter and if there's a problem, it would be reported by disk usage indicators. 
```suggestion

```